mdstat.go

```diff
@@ -0,0 +1,157 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strconv"
+	"strings"
+	"path"
+
+)
+
+var (
+	statuslineRE = regexp.MustCompile(`(\d+) blocks .*\[(\d+)/(\d+)\] \[[U_]+\]`)
+	buildlineRE  = regexp.MustCompile(`\((\d+)/\d+\)`)
+)
+
+type mdStatus struct {
+	mdName       string
```

Comment:
With none of these fields being exported, how are users supposed to the this type?